### PR TITLE
BaSP-MR-161: Activities Form 

### PR DESCRIPTION
**Pull request by IvanJukonis**
PR related to Ticket: 161 which required the implementation of REDUX in order to manipulate the global state of the component. All the requirements has been solved.

### CODE REVIEW

File: src/Components/Activities/FormActivities/index.jsx [hunk 1]
```diff
@@ -2,9 +2,12 @@ import { useState, useEffect } from 'react';
 import style from './modalAdd.module.css';
 import { ModalConfirm, ModalSuccess, Inputs } from '../../Shared';
```
Comment (FrancoLelli) [inline src/Components/Activities/FormActivities/index.jsx:3]:
```suggestion
import { ModalConfirm, ModalSuccess, Inputs, Button } from '../../Shared';
```

File: src/Components/Activities/FormActivities/index.jsx [hunk 2]
```diff
@@ -2,9 +2,12 @@ import { useState, useEffect } from 'react';
...
 import { Button } from '../../Shared/index';
```
Comment (FrancoLelli) [inline src/Components/Activities/FormActivities/index.jsx:4]:
```suggestion

```

File: src/redux/activities/thunks.js [hunk 3]
```diff
@@ -4,21 +4,37 @@ import {
...
+        'Content-Type': 'application/json'
+      }
+    });
+    if (response.ok) {
+      const data = await response.json();
+      const newData = data.data;
+      dispatch(getActivitiesPending(false));
+      dispatch(getActivitiesSuccess(newData));
+    } else {
+      dispatch(getActivitiesPending(false));
```
Comment (FrancoLelli) [inline src/redux/activities/thunks.js:31]:
```suggestion

```

File: src/redux/activities/thunks.js [hunk 4]
```diff
@@ -4,21 +4,37 @@ import {
...
+      dispatch(getActivitiesError(true));
+    }
   } catch (error) {
     dispatch(getActivitiesPending(false));
```
Comment (FrancoLelli) [inline src/redux/activities/thunks.js:35]:
```suggestion

```

**Review by FrancoLelli — APPROVED** [review]
Its OK, but look the comments!!

**Review by CristianLotorto — APPROVED** [review]
nice work!

**Pull request merged**